apps/condo/domains/ticket/hooks/useTableColumns.tsx

```diff
@@ -0,0 +1,314 @@
+import { Tag, Typography, Input, Space, DatePicker, Checkbox } from 'antd'
```

Comment:
hooks it a new folder! It's better to write about it inside `/domains/README.md`

Reply:
Not sure we need to write about each specific folder in domain README